modules/story-budget/story-budget.php

```diff
@@ -322,11 +327,14 @@ function story_budget() {
 	function story_budget_time_range() {
 		
 		$output = '<form method="POST" action="' . menu_page_url( $this->module->slug, false ) . '">';
-			
+
 		$start_date_value = '<input type="text" id="ef-story-budget-start-date" name="ef-story-budget-start-date"'
 			. ' size="10" class="date-pick" value="'
-			. esc_attr( date_i18n( get_option( 'date_format' ), strtotime( $this->user_filters['start_date'] ) ) ) . '" /><span class="form-value">';
-		
+			. esc_attr( date_i18n( get_option( 'date_format' ), strtotime( $this->user_filters[ 'start_date' ] ) ) ) . '" />'
+			. '&nbsp;<input id="ef-story-budget-today-submit" name="ef-story-budget-today-submit" type="submit"'
```

Comment:
This should probably be a link rather than a button?

Reply:
Yea, I'll make it a secondary button for the time being